TicketAir.py

```diff
@@ -0,0 +1,296 @@
+import sqlite3
+import HomePage
+import sys
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+
+class Flight:
+    def __init__(self):
+        self.origin = ''
+        self.destination = ''
+        self.flightNumber = ''
+        self.airplane = None
+        self.date = None
+        self.price = 0
+        self.time = 0
+
+    def __init__(self, origin, destination, flightNumber, plane, date, price, time, sold=0):
+        self.origin = origin
+        self.destination = destination
+        self.flightNumber = flightNumber
+        if isinstance(plane, str):
+            plane = DataBase().getPlane(plane)
+        self.plane = plane
+        self.date = date
+        self.price = price
+        self.time = time
+        self.sold = sold
+
+    def values(self):
```

Comment:
I would refrain from creating public `values` method in your dataclasses. Instead I would recommend creating `add/save`, `remove` etc methods directly inside them (or in base class for data objects) instead of DataBase and also add classmethod for initializing the database table or getting data (flights for example). This way you will have definition of DB fields and DB values closer to each other.